src/Admin.php

```diff
@@ -138,12 +145,12 @@ public static function post_action_label($post_id) {
     }
 
     $formatted_regular_price = wc_price($regular_price, ['price_format' => '%1$s&nbsp;%2$s']);
-    // Replace ",00" with ",-".
-    $formatted_regular_price = preg_replace('@,00@', ',–', $formatted_regular_price);
+    // Replace ",00".
```

Comment:
Now it's no longer clear what is being done here and the definition is very far away.

I'd recommend to use a simple local variable instead of a constant and restore the first comment as it was before and remove the second comment altogether.